pkg/api/v1/clients/kube/cache/cache_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("kube core cache tests", func() {
 			client = kubernetes.NewForConfigOrDie(cfg)
 		})
 
-		Context("all namesapces", func() {
+		Context("all namespaces", func() {
```

Comment:
haha when I was looking at this test, I kept getting bothered by this so thanks for fixing!